profiler/upload_test.go

```diff
@@ -161,13 +188,16 @@ func BenchmarkDoRequest(b *testing.B) {
 
 type testServerWaiter func() (http.Header, map[string]string, []string)
 
-func makeTestServer(t *testing.T, statusCode int) (*httptest.Server, *url.URL, testServerWaiter) {
+type makeServer func(handler http.Handler) (io.Closer, string)
+type startServer func(*testing.T, int) (io.Closer, string, testServerWaiter)
```

Comment:
I would group these
```suggestion
type (
    makeServer func(handler http.Handler) (io.Closer, string)
    startServer func(*testing.T, int) (io.Closer, string, testServerWaiter)
)
```
I'd also document these. They both sound like functions but they're not, they are types.

Reply:
Also where is `startServer` used?

Reply:
it's used in the fixture https://github.com/DataDog/dd-trace-go/pull/788/files#diff-99c478371646b275ba89c2c612bbe68795e2ac000196e31c840382353d5c1890R54

Reply:
Ok I don't think that it warrants a type of its own. It's only used in one place and will be clearer if more verbose in there...